src/editors/sharedComponents/InsertLinkModal/BlockLink/index.test.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
```

Comment:
Do you think we can update the test to avoid the use of Enzyme?  

According to the [OEP 0011](https://github.com/openedx/open-edx-proposals/blob/9ecc51a97d0fd6312fc47de4cd1e153f479f06a9/oeps/best-practices/oep-0011-bp-FED-technology.rst) 

>  New tests must not use Enzyme, and any repositories planning to move to React 18 or newer need to replace Enzyme.

Also, testing-library is already installed in this package.